crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -558,19 +564,31 @@ fn check_duplicates(checker: &mut Checker, values: &Expr) {
     else {
         return;
     };
+    if elts.len() < 2 {
+        return;
+    }
 
     let mut seen: FxHashMap<ComparableExpr, usize> =
         FxHashMap::with_capacity_and_hasher(elts.len(), BuildHasherDefault::default());
+    let mut prev = &elts[0];
     for (index, elt) in elts.iter().enumerate() {
         let expr = ComparableExpr::from(elt);
         seen.entry(expr)
             .and_modify(|index| {
-                checker.diagnostics.push(Diagnostic::new(
+                let mut diagnostic = Diagnostic::new(
                     PytestDuplicateParametrizeTestCases { index: *index },
                     elt.range(),
-                ));
+                );
+                if checker.patch(diagnostic.kind.rule()) {
+                    let del_range = TextRange::new(prev.end(), elt.end());
+                    if !checker.indexer().comment_ranges().intersects(del_range) {
+                        diagnostic.set_fix(Fix::automatic(Edit::range_deletion(del_range)));
```

Comment:
I think this won't quite do the right thing if one of the expressions is parenthesized (the range won't include the parentheses). We need to do something similar to `remove_argument`.